forge-gui/res/cardsfolder/upcoming/park_heights_maverick.txt

```diff
@@ -0,0 +1,12 @@
+Name:Park Heights Maverick
+ManaCost:2 G
+Types:Creature Human Soldier
+PT:2/2
+K:Dethrone
+S:Mode$ CantBlockBy | ValidAttacker$ Creature.Self | ValidBlocker$ Creature.powerLE2 | Description$ CARDNAME can't be blocked by creatures with power 2 or less.
+T:Mode$ ChangesZone | Origin$ Battlefield | Destination$ Graveyard | ValidCard$ Card.Self | Execute$ TrigProliferate | TriggerDescription$ When CARDNAME dies, proliferate.
+T:Mode$ DamageDone | ValidSource$ Card.Self | ValidTarget$ Player | CombatDamage$ True | Execute$ TrigProliferate | TriggerDescription$ Whenever CARDNAME deals combat damage to a player, proliferate.

```

Comment:
Secondary$ True | TriggerDescription$ Whenever Park Heights Maverick deals combat damage to a player or dies, proliferate.`